### ISSUE TITLE
Wrose generated image compare to Convnextv2

### ISSUE THREAD

**Issue by powermano** [keyu-tian/SparK#90]
This is using spark network and replace the final project layer of convnext to Unet decoder. The loss is 0.16 roughly and the vis results is as following:
![test_epoch_30](https://github.com/user-attachments/assets/131b6776-aeb2-48b7-b391-fbc1ba62d256)

This using convnextv2 , the loss is 0.17 roughly
![test_epoch_60](https://github.com/user-attachments/assets/dfc9b3c6-6829-43af-bd63-14294be17d8a)

I think using unet decoder will let model learn some shortcuts for generate the images, as the training loss is almost the same.
